xbmc/platform/linux/input/LibInputSettings.cpp

```diff
@@ -58,7 +58,7 @@ CLibInputSettings::CLibInputSettings(CLibInputHandler *handler) :
   std::set<std::string> settingSet;
   settingSet.insert(SETTING_INPUT_LIBINPUTKEYBOARDLAYOUT);
   settingsManager->RegisterCallback(this, settingSet);
-  settingsManager->RegisterSettingOptionsFiller("libinputkeyboardlayout",
+  settingsManager->RegisterSettingOptionsFiller("input.libinputkeyboardlayout",
```

Comment:
This change looks unrelated to this PR.

Reply:
that was a coincidence catch while diggin in the log, open a separate pr for this? ok

Reply:
@lrusak please feel free to pick this, so i can throw it outta here